plugins/out_stackdriver/stackdriver.h

```diff
@@ -61,10 +64,18 @@ struct flb_stackdriver {
     flb_sds_t token_uri;
     bool metadata_server_auth;
 
-    /* metadata server (GCP specific, WIP) */
-    flb_sds_t zone;
-    flb_sds_t instance_id;
-    flb_sds_t instance_name;
+    /* metadata server url */
+    flb_sds_t metadata_server;
+
+    /*
+     * Stackdriver monitored resource labels
+     * see: https://cloud.google.com/logging/docs/api/v2/resource-list
+     *
+     * project_id is reused from the parsed credentials file and present in all
+     * monitored resources.
+     */
+
```

Comment:
I usually suggest removing the blank line between a comment and the entity it applies to, to keep them together visually and avoid them being accidentally separated even more.

Reply:
Same, oversight on my part.